Content.Shared/_CM14/Weapons/Ranged/CMGunSystem.cs

```diff
@@ -116,7 +116,7 @@ private void OnGunUserWhitelistAttemptShoot(Entity<GunUserWhitelistComponent> en
 
         args.Cancelled = true;
         var gun = Loc.GetString("zzzz-the", ("ent", ent.Owner));
-        _popup.PopupClient($"You don't seem to know how to use {gun}", args.User, args.User);
+        _popup.PopupClient(Loc.GetString("cm-gun-unskilled", ("gun", gun)), args.User, args.User);
```

Comment:
Line above can be removed, see the review for the ftl file.
```suggestion
        _popup.PopupClient(Loc.GetString("cm-gun-unskilled", ("gun", ent.Owner)), args.User, args.User);
```